web/src/lib/components/admin-page/delete-confirm-dialoge.svelte

```diff
@@ -30,15 +36,31 @@
 
 <ConfirmDialogue
   title="Delete User"
-  confirmText="Delete"
+  confirmText={forceDelete ? 'Permanently Delete' : 'Delete'}
   onConfirm={handleDeleteUser}
   onClose={() => dispatch('cancel')}
 >
   <svelte:fragment slot="prompt">
     <div class="flex flex-col gap-4">
-      <p>
-        <b>{user.name}</b>'s account and assets will be permanently deleted after {$serverConfig.userDeleteDelay} days.
-      </p>
+      {#if forceDelete}
+        <p>
+          <b>{user.name}</b>'s account and assets will be queued for permanent deletion <b>immediately</b>.
+        </p>
+      {:else}
+        <p>
+          <b>{user.name}</b>'s account and assets will be scheduled for permanent deletion in {$serverConfig.userDeleteDelay}
+          days.
+        </p>
+      {/if}
+
+      <div class="flex justify-center m-4 gap-2">
+        <label class="text-sm dark:text-immich-dark-fg" for="forceDelete">
+          Queue user and assets for immediate deletion
+        </label>
+
+        <input id="forceDelete" type="checkbox" class="form-checkbox h-5 w-5 color" bind:checked={forceDelete} />
```

Comment:
Accidentally deleting a user is still a little too easy in my opinion, even with this extra checkbox. How about requiring an additional email input for permanent deletions, kinda like github does when deleting a repository?

Also the `color` class doesn't exist

Reply:
Alex had this same idea and I think it is a good one. I don't care if it is done now or in a separate pull request.

Reply:
I should have time this weekend to work on that - can do it here or in a new PR, doesn't matter to me either. You thinking an input box that appears when you check the box that disables the button until the user id is entered? Or a separate screen like github that shows up after you click delete that then asks you input?

Reply:
Added the extra input logic for force delete

Reply:
Updated screenshots, the force checkbox jumping when enabling it may be a bit disorienting

Reply:
Perfect. Looks great